content/doc/pipeline/declarative/Advanced.adoc

```diff
@@ -0,0 +1,105 @@
+The pipeline you have seen so far has been mostly declarative. You can
+do a lot more with pipeline (in fact there isn't really anything you
+can't do, if you are careful).
+
+[[using-pipeline-scripting-inside-stages]]
+Using pipeline scripting inside stages
+--------------------------------------
```

Comment:
@rtyler - Do we care about the section heading styling?  So far, I've seen `== Using pipeline scripting inside stages` for example.